guest-os-info.md

```diff
@@ -0,0 +1,49 @@
+## Guest Operating System Information
+
+Guest operating system identity for the Virtual Machine will be provided by the label ``os-short-id`` :
+
+```
+metadata:
+  name: myvm
+  labels:
+    os-short-id: win2k12r2
+```
+
+The ``os-short-id`` label is based on the short OS identifier from [libosinfo](https://libosinfo.org/).
+Libosinfo database can be queried to extract additional information about the operating system using the short-id:
+
+```
+# List all operating systems
+$ osinfo-query os
+
+ Short ID             | Name                                              | Version | ID
+----------------------------------------------------------------------------------------------------------------------
+...
+ fedora24             | Fedora 24                                      | 24        | http://fedoraproject.org/fedora/24
+ fedora25             | Fedora 25                                      | 25        | http://fedoraproject.org/fedora/25
+ fedora26             | Fedora 26                                      | 26        | http://fedoraproject.org/fedora/26
+...
+ win2k12r2          | Microsoft Windows Server 2012 R2 | 6.3       | http://microsoft.com/win/2k12r2
+...
+```
+
+```
+# Get OS specific info
+$ osinfo-query os short-id=win2k12r2
+
+ Short ID             | Name                                              | Version | ID
+----------------------------------------------------------------------------------------------------------------------
+ win2k12r2          | Microsoft Windows Server 2012 R2 | 6.3       | http://microsoft.com/win/2k12r2
+
+```
+
+Addtional properties can be queried include : name, version, family, vendor, release-date, eol-date, codename, id.
```

Comment:
You say about ability to query. How one could do it? Please make sure to include this info here.